go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -204,10 +197,8 @@ func (c *addressManager) runInternal(stopChan <-chan struct{}, doneWg *sync.Wait
 	klog.Info("Node IP manager is running")
 }
 
-// handleNodeChanges takes a node obj, extracts its name and determines if the node's address changed. If so, it
-// updates the node's OVS encapsulation IP.
-func (c *addressManager) handleNodeChanges(obj interface{}) {
-	var err error
+// handleNodeChanges updates OVN's EncapIP if the node IP changed
+func (c *addressManager) handleNodeChanges() {
```

Comment:
Should we rename this function as handleNodePrimaryAddrChange? It is referenced in other calls which are irrelevant to node object change.

Maybe create a new func: handleNodePrimaryAddrChange and have handleNodeChanges to call it? wdyt

Reply:
I will just rename it since there is no reason to have another function that takes in an `obj` without it doing anything with it.